libraries/botbuilder-dialogs-adaptive/src/skillExtensions.ts

```diff
@@ -24,7 +34,7 @@ export class SkillExtensions {
     /**
      * Configures the skill conversation id factory to use.
      */
-    public static useSkillConverationIdFactory(dialogManager: DialogManager, skillConversationIdFactory: SkillConversationIdFactoryBase): DialogManager {
+    public static useSkillConversationIdFactory(dialogManager: DialogManager, skillConversationIdFactory: SkillConversationIdFactoryBase): DialogManager {
```

Comment:
Can you add the parameters to the docstring?

Reply:
OK, I'll add that.